bedrock/base/templates/includes/banners/vpn-coupon-variations/vpn-coupon-promo-1.html

```diff
@@ -0,0 +1,58 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% if LANG == 'de' %}
+  {% set banner_title = "Sicher durchs Netz" %}
+  {% set banner_subtitle = "Mit 20 % Rabatt aufs Jahresabo!" %}
+  {% set banner_text = "Nutze jetzt den code <span class='c-coupon-code'>VPN20</span> beim checkout" | safe %}
+  {% set link_text = "Jetzt sparen" %}
+{% elif LANG == 'fr' %}
+  {% set banner_title = "Voyagez en toute sécurité" %}
+  {% set banner_subtitle = "20% sur l’abonnement d’un an" %}
+  {% set banner_text = "Entrez le code <span class='c-coupon-code'>VPN20</span> au paiement" | safe %}
+  {% set link_text = "J’en profite" %}
+{% else %}
+  {% set banner_title = "Safe travels" %}
+  {% set banner_subtitle = "20% off yearly plan" %}
+  {% set banner_text = "Use code <span class='c-coupon-code'>VPN20</span> at checkout" | safe%}
+  {% set link_text = "Try now" %}
+{% endif %}
+
+{% set query_param = "?entrypoint_experiment=vpn-coupon-promo-banner&entrypoint_variation=1" %}
+
+<aside class="c-banner hide-from-legacy-ie" id="vpn-coupon-promo-banner">
+  <div class="c-banner-inner">
+    <div class="c-coupon-wrapper c-coupon">
+      <div class="c-coupon-title-wrapper">
+        <h2 class="c-coupon-title">
+          {{ banner_title|safe }}
+        </h2>
+      </div>
+      <div class="c-coupon-content-wrapper">
+        <h3 class="c-coupon-subtitle">
+          {{ banner_subtitle|safe }}
+        </h3>
+        <p class="c-coupon-text">
+          {{ banner_text|safe }}
+        </p>
+        <div class="c-cta">
+            {{ vpn_product_referral_link(
+              referral_id='vpn-coupon-promo-banner',
+              link_text=link_text,
+              page_anchor= query_param + '#pricing',
+              class_name='mzp-t-secondary',
+              optional_attributes= {
+                  'data-cta-text' : link_text | safe ,
+                  'data-cta-type' : 'link',
+                  'data-cta-position' : 'banner',
+              }
+          ) }}
```

Comment:
This helper is meant for linking to the VPN landing page from another page, and as such creates an absolute link which is overriding the query parameters here. It also means the `referral_id` will be ignored here.

Is there a reason why you needed to use this helper here, as opposed to a simple `<a href="#pricing">Try now</a>` link?